condition-checker/src/main/java/com/mersey/rowing/club/condition_checker/controller/ConditionController.java

```diff
@@ -9,8 +11,14 @@
 @Slf4j
 public class ConditionController {
 
+    @Autowired
+    private OpenWeatherApiClient openWeatherApiClient;
+
     @GetMapping("/get_conditions")
     public void getConditions(@RequestHeader(value = "date", required = false) String date, @RequestHeader(value = "time", required = false) String time){
+
```

Comment:
actually not using here, so feel free to remove these from this file for now, although it will also be added back in in the next PR so completely up to you